iidm/iidm-modification/src/test/java/com/powsybl/iidm/modification/ShuntCompensatorModificationTest.java

```diff
@@ -82,4 +83,20 @@ void testConnectionAndPositionChange() {
         Assertions.assertEquals(1, shunt.getSectionCount());
     }
 
+    @Test
+    void testConnectionOnRegulatingTerminal() {
```

Comment:
As for the generators, could you add tests where no other regulating element is found?